core/pactbroker/src/main/kotlin/au/com/dius/pact/core/pactbroker/PactBrokerClient.kt

```diff
@@ -157,6 +157,20 @@ interface IPactBrokerClient {
     version: String
   ): Result<Boolean, List<String>>
 
+  /**
+   * Publish provider branch to the Pact broker
+   * @param docAttributes Attributes associated with the fetched Pact file
+   * @param tag Provider name
```

Comment:
old docs?

Reply:
+1, I fixed it in the tags method as well